js/contributors.js

```diff
@@ -1,53 +1,66 @@
-window.onload = function(){
-  $.ajax({
-    url: "https://api.github.com/repos/fossasia/labyrinth/contributors"
-  }).done(function(data){
-    data.forEach(function(contributors){
-      var html = '<div class="column"><div class="card">';
-      html += '<div class="contributorsImage">';
-      html += '<img src="https://github.com/'+contributors.login+'.png?size=300x300" style="width:100%">';
-      html += '</div>';
-      html += '<br>';
-      html += '<div class="lol"><p>'+contributors.login+' ('+contributors.contributions;
-      if (contributors.contributions === 1) {
-        html += ' contribution)</p>';
-      } else {
-        html += ' contributions)</p>';
-      }
-      html += '<p class="middle"><button class="button"> ';
-      html += '<a href="'+contributors.html_url+'""> View on Github';
-      html += '</button></p>';
-      html += '</a></div></div></div>';
-      $("#contributors").append(html);
+window.onload = function () {
+    $.ajax({
```

Comment:
Please see the style guide. https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md#code-style-recommendations We use two spaces in all documents. Changing back and forth makes it harder to read the history and understand the change. What do you think?

Reply:
@niccokunzmann I wrote the code with two spaces but I think my editors auto corrected it to tabspace. I will correct it. Thanks for pointing that out.